### PR TITLE
update develop to v0.20.3

### PR DESCRIPTION
**Pull request by theguy951357**
## Describe your changes
updating develop to v0.20.3
## Provide Issue ticket number if applicable/not in title
no ticket number
## Provide testing steps
no testing needed
## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [x] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [x] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [x] Netclient & Netmaker are awesome.

### CODE REVIEW

**Review by mattkasun — APPROVED** [review]

**Review by Aceix — CHANGES_REQUESTED** [review]
can the gui version be updated as well. check `gui/main.go` file

**Comment by abhishek9686** [comment]
> can the gui version be updated as well. check `gui/main.go` file

will this be in a separate PR?

**Comment by theguy951357** [comment]
> 

updated

File: gui/main.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ var assets embed.FS
 
 var appIcon = GetFileAsBytes("./appicon.png")
 
-var version = "v0.18.8"
+var version = "v0.22.3"
```
Comment (abhishek9686) [inline gui/main.go:23]: supposed to be `v0.20.3`?

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Review by Aceix — APPROVED** [review]

**Pull request merged**